### PR TITLE
Prevent notifying each time

### PR DESCRIPTION
**Pull request by eisuke**
Prevent notifying the message each time when accessed the already completed job_instance path.

@cookpad/dev-infra please review.

### CODE REVIEW

**Comment by hogelog** [comment]
> Prevent notifying the message each time when accessed the already completed job_instance path.

This reason cannot guess from this code.
Maybe you should comment it.

**Comment by eisuke** [comment]
> But I like more simply implementation.

I agree. I redoed it f9db129

File: app/assets/javascripts/kuroko2/job_instances.js [hunk 1]
```diff
@@ -45,6 +48,7 @@ jQuery(function ($) {
         tbody.data('last-log-id', data.logs[data.logs.length - 1].id);
       }
 
+      $('#logs table').data("reload", data.reload);
```
Comment (eagletmt) [inline app/assets/javascripts/kuroko2/job_instances.js]: What is this needed for?

Comment (eisuke) [inline app/assets/javascripts/kuroko2/job_instances.js]: It is used the below line.
https://github.com/eisuke/kuroko2/blob/f9db129f58e3d045db2356f0380fe27a69d1153c/app/assets/javascripts/kuroko2/job_instances.js#L90

Comment (eagletmt) [inline app/assets/javascripts/kuroko2/job_instances.js]: Okay, but I couldn't figure out why and how this is related to "Prevent notifying each time" ...

Comment (eisuke) [inline app/assets/javascripts/kuroko2/job_instances.js]: Good point...  it will fix later 33ec571

**Pull request merged**